### ISSUE TITLE
Querying and downloading images pop up windows, can they be incorporated into main view window?

### ISSUE THREAD

**Issue by kwalz** [mbari-org/vars-gridview#73]
These two windows are small and disappear behind applications, would be nice to incorporate these tasks as status bars on the main Gridview window rather than pop up windows that are hard to locate with a lot of windows open on a computer.
<img width="201" alt="Screenshot 2024-01-18 at 3 31 29 PM" src="https://github.com/mbari-org/vars-gridview/assets/34140155/aecad4f7-58d8-47f9-9e4e-97c8de9212ee">

**Comment by kevinsbarnard** [comment]
@kwalz Can do!

**Comment by lonnylundsten** [comment]
@kwalz @kevinsbarnard Is this done?

**Comment by kwalz** [comment]
@lonnylundsten No, this wasn't done, we decided to implement this in an updated version of Gridview, we tabled it to that new version and Kevin moved it to **ToDo** and added **GUI** in label (see above). We will want to keep all these notes for an updated Gridview.
